### DISCUSSION TITLE
Ubuntu disappeared from my VS Code and Terminal

### DISCUSSION THREAD

**Discussion by tsekeres** [nss-evening-cohort-14/evening-client-side#115] [Help Me!] [answered]
Your Name: Tad S

Have you searched the other help tickets?Yes and sadly most of the ones that popped up are from me.

If there are similar issues, make sure to give us their number/link to them and tell how yours is different. Not really

Describe your issue: Ubuntu disappeared from my options for terminal use. Its not in the dropdown in my terminal and not an option in the VS Code terminal selection processs.

List any error messages that appear.N/A

What have you googled?
https://askubuntu.com/questions/1086509/ubuntu-18-terminal-not-opening
https://ubuntu.com/tutorials/ubuntu-on-windows#1-overview
https://www.maketecheasier.com/fix-ubuntu-cannot-open-terminal/

What have you tried? Tell us everything you have tried.
Most of the web research had to do with a, "Ubuntu Terminal", opening my terminal with Ubuntu or  with Python issues. 
I went back through the install process from January. The Ubuntu entry is still in the terminal settings. It just doesn't show up in the dropdown anymore. The Bash terminal option still works so I guess I could just change over to that. Halp!

**Comment by drteresavasquez** [comment]
Yes. Convert to bash. After going through the installs I realized that the day cohort uses the bash terminal so that is the one I would recommend as it works and the help you can get for us would likely center around that. 

If bash works go for that one instead.

**Comment by tsekeres** [comment, marked as answer]
Converting to Bash!
